makedist

```diff
@@ -34,6 +34,7 @@ usage()
     (
     echo "Usage: makedist [-s] [ -m machinetype ] {osg|egi|default|none}"
     echo "       makedist [-s] -o <self_extracting_script>"
+    echo "       makedist [-v] X.Y.Z"
```

Comment:
No, this should be included as `[-v X.Y.Z]` in the first Usage line.  It doesn't work by itself or as part of `-o`.

Reply:
While you're there please remove the blanks surrounding `-m machinetype`.

Reply:
No, not on a line by itself.  Include it in the first Usage line.